src/mechanics/cell/Cell.h

```diff
@@ -58,6 +58,35 @@ class Cell : public CellInterface
         return mId;
     }
 
+    Eigen::VectorXd Interpolate(Eigen::VectorXd naturalCoords) const override
+    {
+        const auto& coordinateElement = mElements.CoordinateElement();
+        return coordinateElement.GetNMatrix(naturalCoords) * coordinateElement.ExtractNodeValues();
```

Comment:
There is the free function `Interpolate(ElementInterface, NaturalCoords)` for that.